src/streamlit_extras/altex/__init__.py

```diff
@@ -248,6 +232,8 @@ def chart(use_container_width: bool = True, **kwargs):
 
 @st.experimental_memo
 def example_line():
+    from streamlit_extras.altex import line_chart
```

Comment:
Adding this explicit import since default docstring in the gallery would solely give
```
from streamlit_extras.altex import chart
```
where `chart` is the extra's unique function. 

Will probably add some logic to not display the `from streamlit_extras.altex import chart` if there's an explicit import already... but that's also very hacky

Reply:
Seems like it might be time to address the "what if a library has more than one function" elephant

Reply:
Right...

Reply:
If you wait for #61, this will get much easier :)

Reply:
Done!